scout/build/variant/variant.py

```diff
@@ -416,6 +424,13 @@ def add_frequencies(variant_obj, frequencies):
     variant_obj["clingen_ngi"] = frequencies.get("clingen_ngi")
     variant_obj["swegen"] = frequencies.get("swegen")
 
+    # add the mei frqs:
```

Comment:
What about setting these variables in a loop:

for freq in ["swegen_alu", "swegen_herv" ...]:
   variant_ob[freq] = frequencies.get(freq)